src/UnwindRustSgx.c

```diff
@@ -0,0 +1,150 @@
+#define _GNU_SOURCE
+#include <link.h>
+
+#include <elf.h>
+#include <stdarg.h>
+#define max_log 256
+#include "UnwindRustSgx.h"
+
+#ifdef __cplusplus
+extern "C"  {
+#endif
+	int c_rwlock_rdlock(RWLock *rwlock);
+	int c_rwlock_wrlock(RWLock *rwlock);
+	int c_rwlock_unlock(RWLock *rwlock);
+	unsigned char *__rust_alloc(size_t, size_t);
+	void __rust_dealloc(unsigned char *, size_t);
+	int c_print_msg(uint8_t *m, int s);
+	void c_abort(void);
+	unsigned char *c_encl_address(size_t);
+
+#ifdef __cplusplus
+}
+#endif
+
+void abort() __attribute__((noreturn)) {
```

Comment:
For the following functions:
* abort
* libuw_pthread_rwlock_rdlock
* libuw_pthread_rwlock_wrlock
* libuw_pthread_rwlock_unlock
* __libuw_stack_chk_fail

Can you just use `#define` instead of writing all these forwarding implementations? Or just name the functions correctly in Rust?